nobl9/resource_direct_test.go

```diff
@@ -134,6 +134,16 @@ resource "nobl9_direct_%s" "%s" {
   source_of = ["Metrics", "Services"]
   url = "https://web.net"
   dynatrace_token = "secret"
+  historical_data_retrieval {
+    default_duration  {
+      unit = "Day"
+      value = 0
+    }
+    max_duration {
+      unit = "Day"
+      value = 0
+    }
+  }
```

Comment:
Can we add at 222 line `  insights_query_key = "NRIQ-secret"` because in meantime there was change in validator of that secret. It breaks tests.